install-deps.sh

```diff
@@ -370,7 +370,7 @@ else
                     $SUDO dnf config-manager --set-enabled PowerTools
 		    # before EPEL8 and PowerTools provide all dependencies, we use sepia for the dependencies
 		    $SUDO dnf config-manager --add-repo http://apt-mirror.front.sepia.ceph.com/lab-extras/8/
-		    $SUDO dnf config-manager --setopt gpgcheck=0 apt-mirror.front.sepia.ceph.com_lab-extras_8_ --save
+		    $SUDO dnf config-manager --save --setopt=apt-mirror.front.sepia.ceph.com_lab-extras_8_.gpgcheck=0
```

Comment:
personally I would avoid reordering the options, lest someone think that's relevant

Reply:
yeah, i was thinking, well, change the option and save it, that was the reason i put `save` at last. but if this addresses the failure i am all for it.

Reply:
That's the point, moving --save doesn't affect the fix; the fix is to add the repoid.  Not moving --save avoids the confusion I just warned about and you just fell into.

Reply:
I moved save due to the ordering used by the dnf developer here, https://bugzilla.redhat.com/show_bug.cgi?id=1702678 I considered it a little insurance against subsequent failure, but I don't mind if your adamant.

Reply:
Changed my mind since the RH internal test puts the '--save' at the end.